### ISSUE TITLE
Pinned version of async-trait conflicts with other dependencies

### ISSUE THREAD

**Issue by oddgrd** [libsql/libsql-client-rs#22]
Hey! I was just testing out turso (very cool by the way!) with an axum app, and ran into an issue due to your [pinned version](https://github.com/libsql/libsql-client-rs/commit/6ba07d29d28cc6dbf3ab1c48b8e22b8362ae3f28) of async-trait. Is there any chance you could un-pin it, or did you encounter a bug on higher versions?

Error message:
```
Error: failed to get cargo metadata

Caused by:
    `cargo metadata` exited with an error:     Updating crates.io index
    error: failed to select a version for `async-trait`.
        ... required by package `axum v0.6.18`
        ... which satisfies dependency `axum = "^0.6.18"` of package `turso-axum v0.1.0 (/home/oddgrd/dev/shuttle/examples/axum/turso-axum)`
    versions that meet the requirements `^0.1.67` are: 0.1.68, 0.1.67

    all possible versions conflict with previously selected packages.

      previously selected package `async-trait v0.1.56`
        ... which satisfies dependency `async-trait = "=0.1.56"` of package `libsql-client v0.28.0`
        ... which satisfies dependency `libsql-client = "^0.28.0"` of package `turso-axum v0.1.0 (/home/oddgrd/dev/shuttle/examples/axum/turso-axum)`

    failed to select a version for `async-trait` which could resolve this conflict
   ``

**Comment by psarna** [comment]
ah, sure! Will try, as I don't even remember exactly why it was pinned in the first place. Let me try to cook a quick patch

**Comment by psarna** [comment]
argh, and first and foremost, a while ago I got rid of async-trait altogether, so this dependency is redundant... I'll just drop it in a minute

**Comment by psarna** [comment]
Just published 0.29 without the outdated dependency, thanks for reporting!

**Comment by oddgrd** [comment]
That was fast, thanks!
